### PR TITLE
remove percentages

### PR DESCRIPTION
**Pull request by ocefpaf**
Exactly what is says above :smile:

### CODE REVIEW

**Comment by ocefpaf** [comment]
I'm also ignoring "W504 line break after binary operator" rule. Even though pep8 recommends it:

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +34,6 @@ script:
       fi
 
     - if [[ $TEST_TARGET == 'coding_standards' ]]; then
-        pycodestyle --ignore=E741,E226 --max-line-length=100 tests ;
-        pycodestyle --ignore=E741,E226 --max-line-length=100 utide ;
+        pycodestyle --ignore=E741,E226,W504 --max-line-length=100 tests ;
```
Comment (wesleybowman) [inline .travis.yml]: `pycodestyle`, by default, ignores:
E121, E123, E126, E133, E226, E241, E242, E704, W503, W504

I think this is true, unless you specify the `ignore` flag. Is there any others from that list that we want to include?

File: README.rst [hunk 2]
```diff
@@ -24,19 +24,18 @@ Still in heavy development--everything is subject to change!
...
-    % For more information see:
-    % Codiga, D.L., 2011. Unified Tidal Analysis and Prediction Using the
-    % UTide Matlab Functions. Technical Report 2011-01. Graduate School
-    % of Oceanography, University of Rhode Island, Narragansett, RI.
-    % 59pp. ftp://www.po.gso.uri.edu/pub/downloads/codiga/pubs/
-    % 2011Codiga-UTide-Report.pdf
-    %
-    % UTide v1p0 9/2011 d.codiga@gso.uri.edu
-    % http://www.po.gso.uri.edu/~codiga/utide/utide.htm
+    Codiga, D.L., 2011. Unified Tidal Analysis and Prediction Using the
```
Comment (wesleybowman) [inline README.rst]: Nice. I should have done this at the beginning, but I wanted to copy the matlab style 😆

File: .travis.yml (hunk 1 above)
Comment (ocefpaf) [reply .travis.yml]: I don't think so. To be honest some of those I would like to remove from that list, like W504. But that is low priority and I don't think we should worry about it now.

File: README.rst (hunk 2 above)
Comment (ocefpaf) [reply README.rst]: I still have tons of code with that kind of copy-n-paste :smile:

**Comment by wesleybowman** [comment]
@ocefpaf I will merge now (I always forget you don't self merge 😛 )

**Comment by ocefpaf** [comment]
Thanks @wesleybowman!

**Pull request merged**